salt/metalk8s/volumes/prepared/files/metalk8s-volume-clean-up

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import os
+import subprocess
+import sys
+
+
+def die(message):
+    print(message, file=sys.stderr)
+    sys.exit(1)
+
+
+def get_volume_id():
+    if len(sys.argv) != 2:
+        die("Invalid number of arguments: expected only one volume ID")
+
+    return sys.argv[1]
+
+
+def find_device(volume_id):
+    path_by_uuid = "/dev/disk/by-uuid/{}".format(volume_id)
+    if os.path.exists(path_by_uuid):
+        return path_by_uuid
```

Comment:
Could actually read the symlink here to always return `/dev/loopN` from this function.

Reply:
Done.